File: CS/CalculatedPropertiesSolution.Module/DatabaseUpdate/Updater.cs
using System;
using System.Linq;
using DevExpress.ExpressApp;
using DevExpress.Data.Filtering;
using DevExpress.Persistent.Base;
using DevExpress.ExpressApp.Updating;
using DevExpress.Xpo;
using DevExpress.ExpressApp.Xpo;
using DevExpress.Persistent.BaseImpl;
using CalculatedPropertiesSolution.Module.BusinessObjects;

namespace CalculatedPropertiesSolution.Module.DatabaseUpdate {
    public class Updater : ModuleUpdater {
        public Updater(IObjectSpace objectSpace, Version currentDBVersion) :
            base(objectSpace, currentDBVersion) {
        }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            Product product = ObjectSpace.CreateObject<Product>();
            product.Name = "Chai";
            for (int i = 0; i < 10; i++) {
                Order order = ObjectSpace.CreateObject<Order>();
                order.Product = product;
                order.Description = "Order " + i.ToString();
                order.Total = i;
            }

            ObjectSpace.CommitChanges();
        }
        public override void UpdateDatabaseBeforeUpdateSchema() {
            base.UpdateDatabaseBeforeUpdateSchema();
        }
    }
}

File: VB/CalculatedPropertiesSolution.Module/DatabaseUpdate/Updater.vb
Imports System
Imports System.Linq
Imports DevExpress.ExpressApp
Imports DevExpress.Data.Filtering
Imports DevExpress.Persistent.Base
Imports DevExpress.ExpressApp.Updating
Imports DevExpress.Xpo
Imports DevExpress.ExpressApp.Xpo
Imports DevExpress.Persistent.BaseImpl
Imports CalculatedPropertiesSolution.Module.BusinessObjects

Namespace CalculatedPropertiesSolution.Module.DatabaseUpdate
	Public Class Updater
		Inherits ModuleUpdater

		Public Sub New(ByVal objectSpace As IObjectSpace, ByVal currentDBVersion As Version)
			MyBase.New(objectSpace, currentDBVersion)
		End Sub
		Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
			MyBase.UpdateDatabaseAfterUpdateSchema()
			Dim product As Product = ObjectSpace.CreateObject(Of Product)()
			product.Name = "Chai"
			For i As Integer = 0 To 9
				Dim order As Order = ObjectSpace.CreateObject(Of Order)()
				order.Product = product
				order.Description = "Order " & i.ToString()
				order.Total = i
			Next i

			ObjectSpace.CommitChanges()
		End Sub
		Public Overrides Sub UpdateDatabaseBeforeUpdateSchema()
			MyBase.UpdateDatabaseBeforeUpdateSchema()
		End Sub
	End Class
End Namespace
